ui-core/src/types/study.ts

```diff
@@ -68,7 +68,10 @@ export type NotificationConfig = {
 
 export type EmailTemplate = {
   subject: string
-  body: string
+  body: string,
+  name: string,
+  stableId: string,
```

Comment:
Nit: for consistency, none of the other types in this file have trailing commas.

```suggestion
  body: string
  name: string
  stableId: string
```